pom.xml

```diff
@@ -4,7 +4,7 @@
 
 	<groupId>com.github.magpiebridge</groupId>
 	<artifactId>magpiebridge</artifactId>
-	<version>0.1.4-SNAPSHOT</version>
+	<version>0.2.0-SNAPSHOT</version>
```

Comment:
Why are you suggesting 0.2.0-SNAPSHOT? The next version would be 0.1.5.  Can you change it to 0.1.5-SNAPSHOT? I will make a release to maven central.

Reply:
changed the version to v1.5.0 ✅